src/gui/widgets/entry_list.py

```
@@ -0,0 +1,188 @@
+from PyQt6.QtWidgets import (
+    QListWidget, QListWidgetItem, QMenu, QAbstractItemView,
+    QMessageBox, QVBoxLayout, QLabel, QWidget, QHBoxLayout
```

Ruff (0.8.2)

`PyQt6.QtWidgets.QMessageBox` imported but unused

Remove unused import: `PyQt6.QtWidgets.QMessageBox`

(F401)